DeeplinkPlugin/deeplink_service.h

```diff
@@ -8,6 +8,7 @@
 #import <UIKit/UIKit.h>
 #import "app_delegate.h"
 #import "godot_app_delegate.h"
+#import "core/version_generated.gen.h"
```

Comment:
Without importing this file, the `VERSION_MAJOR` and `VERSION_MINOR` flags were not recognized.